packages/web/src/components/ui/CardList/CardList.tsx

```diff
@@ -2,18 +2,26 @@ import React from 'react';
 import styles from './CardList.module.scss';
 import Card, { CardProps } from '../Card/Card';
 import clsx from 'clsx';
+import Skeleton from '../Skeleton/Skeleton';
+import { repeat } from 'utils/helpers';
 
 type Props = {
   cards: CardProps[];
   variant?: 'default' | 'vertical';
+  loading?: boolean;
 };
 
-export default function CardList({ cards, variant = 'default' }: Props) {
+export default function CardList({ cards, variant = 'default', loading }: Props) {
   return (
     <div className={clsx(styles.grid, styles[variant])}>
-      {cards.map((card) => (
-        <Card key={card.id} {...card} />
-      ))}
+      {loading
```

Comment:
@spalt08 @ctizen need your opinion on this one. I tend to favor splitting off skeletons for atom/molecule level components off into a separate component and having loading states only for organism-level components. What direction would we want to go with this one?

Reply:
I also tend to split off skeletons, since the code of actual components looks much cleaner

Reply:
I see no evil in both options at the moment, as the suggestion doesn't change much in total. E.g. splitting skeletons from here will lead to style duplication somewhere else and thus make it less clean. The complexity is either here or there, we can't just ignore it.

Reply:
@KnoWabLe what's your opinion on restructuring skeletons like follows
```
/Component
  Component.scss
  Component.tsx
  ComponentSkeleton.tsx
```